common/djangoapps/enrollment/views.py

```diff
@@ -259,100 +281,167 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
-              'honor'. Only server-to-server requests can enroll with other modes. Optional.
+              * user: The username of the currently logged in user. Optional.
+                You cannot use the command to enroll a different user.
 
-            * is_active: A Boolean indicating whether the enrollment is active. Only server-to-server requests are
-              allowed to deactivate an enrollment. Optional.
+              * mode: The course mode for the enrollment. Individual users
+                cannot upgrade their enrollment mode from 'honor'. Only
+                server-to-server requests can enroll with other modes.
+                Optional.
 
-            * course details: A collection that contains:
+              * is_active: A Boolean value indicating whether the enrollment
+                is active. Only server-to-server requests are allowed to
+                deactivate an enrollment. Optional.
 
-                * course_id: The unique identifier for the course.
+              * course details: A collection that includes the following
+                information.
 
-            * email_opt_in: A Boolean indicating whether the user
-              wishes to opt into email from the organization running this course. Optional.
+                  * course_id: The unique identifier for the course.
 
-            * enrollment_attributes: A list of dictionary that contains:
+              * email_opt_in: Optional. A Boolean value that indicates whether
+                the user wants to receive email from the organization that runs
+                this course.
 
-                * namespace: Namespace of the attribute
-                * name: Name of the attribute
-                * value: Value of the attribute
+              * enrollment_attributes: A list of dictionary that contains:
 
-        **Response Values**
+                  * namespace: Namespace of the attribute
+                  * name: Name of the attribute
+                  * value: Value of the attribute
 
-            A collection of course enrollments for the user, or for the newly created enrollment.
-            Each course enrollment contains:
+              * is_active: Optional. A Boolean value that indicates whether the
+                enrollment is active. Only server-to-server requests can
+                deactivate an enrollment.
 
-                * created: The date the user account was created.
+              * mode: Optional. The course mode for the enrollment. Individual
+                users cannot upgrade their enrollment mode from "honor". Only
+                server-to-server requests can enroll with other modes.
+              
+              * user: Optional. The user ID of the currently logged in user. You
+                cannot use the command to enroll a different user.
 
-                * mode: The enrollment mode of the user in this course.
```

Comment:
@clintonb Can you review the following GET and POST Response Values sections (lines 360-380) and let me know if they're accurate?